pkg/operation/hybridbotanist/cloud_config.go

```diff
@@ -78,10 +78,11 @@ func (b *HybridBotanist) generateCloudConfigChart() (*chartrenderer.RenderedChar
 			},
 			"version": b.Shoot.Info.Spec.Kubernetes.Version,
 		},
-		"workers": workers,
+		"workers":    workers,
+		"kubeconfig": b.Secrets["cloud-config-downloader"].Data["kubeconfig"],
 	}
 
-	config, err = b.InjectImages(config, b.ShootVersion(), b.ShootVersion(), common.HyperkubeImageName)
+	config, err = b.InjectImages(config, b.ShootVersion(), b.ShootVersion(), common.RubyImageName, common.HyperkubeImageName)
```

Comment:
Just for sake of integrity, can you add the RubyImage to https://github.com/gardener/gardener/blob/90c0d6aa6150f3a265dcb5b77b2426a2f9947817/charts/shoot-cloud-config/charts/original/values.yaml?